pkg/ns/ns.go

```diff
@@ -72,30 +73,35 @@ func getCurrentThreadNetNSPath() string {
 	return fmt.Sprintf("/proc/%d/task/%d/ns/net", os.Getpid(), unix.Gettid())
 }
 
-// Returns an object representing the current OS thread's network namespace
+// GetCurrentNS returns an object representing the current OS thread's network namespace
 func GetCurrentNS() (NetNS, error) {
 	return GetNS(getCurrentThreadNetNSPath())
 }
 
 const (
+	// The following are inherited by
 	// https://github.com/torvalds/linux/blob/master/include/uapi/linux/magic.h
+	// hence won't follow golint rules for constants.
 	NSFS_MAGIC   = 0x6e736673
 	PROCFS_MAGIC = 0x9fa0
 )
 
-type NSPathNotExistErr struct{ msg string }
```

Comment:
Hm - this is an API breaking change for this library - we should probably avoid doing that.